### ISSUE TITLE
A11y: The WinUI app window cannot be control using the keyboard while focus is inside a Webview2 control 

### ISSUE THREAD

**Issue by rperez030** [MicrosoftEdge/WebView2Feedback#2961]
**Description**

Keystrokes such as Alt F4 and Alt +Space bar which are needed to control the app window using the keyboard does not propagate to the underlying window while focus is inside the webView2 control. this creates a situation in which a keyboard only user will have to somehow drive focus away from the web view control to be able to close the application or interact with the application window.

**Version**
SDK:  1.2.220902.1-preview1
Runtime:  Microsoft Edge Version 107.0.1418.42 (Official build) (64-bit)
Framework:  WinUI3
OS:  Edition	Windows 11 EnterpriseVersion	22H2 OS build	22621.819 Experience	Windows Feature Experience Pack 1000.22636.1000.0

**Repro Steps**

1. Run the sample app at [https://github.com/rperez030/WinUIExample](https://github.com/rperez030/WinUIExample).
2. Navigate to the WebView2 control using the Tab key and verify the visible focus is inside the web content.
3. Press Alt +F4 to close the application.

#### Expected behavior

The sample app closes and focus returns to the previous app or to the desktop.

#### Current behavior

Nothing happens.

**Screenshots**

**Additional context**

Pressing alt + Space bar to open the control menu which would allow the keyboard user to resize the window or to close it does not work either while focus is inside the web view control. By default, these keystrokes should be allowed to pass to the underlying window.

CC @champnic .

**Comment by champnic** [comment]
Thanks for opening as a new issue and adding these details. I've added this as a bug on our backlog and we'll take a look!

**Comment by akash07k** [comment]
@champnic  Any update on this and on #2330 as well?
Both of these issues are not since fixed since 8 months now and sadly we are having very bad experience as such.
Not only `Alt+F4`, in fact if we once enter into the WebView, we can't go come outside of it by any pressing `Shift+Tab` or any other way.
Not only this, accessibility is still a big mess inside the WebViews with both JAWS, NVDA as well as Narrator.
Screen readers just read few controls via `Tab` key and after they stop recognizing any of the controls.
Only the Browse/Scan mode works which is not a good experience.
I really urge and request Microsoft that they seriously consider these both issues on priority and fix them.
Most of my apps based on WebView2 are completely unusable since all these months and I have some more full stack apps which I want to develop and I am unable to do so due to broken WebViews.
Please do not take these issues lightly. 🙏🙏🙏🙏🙏🙏

**Comment by akash07k** [comment]
is any fix to this planned anytime soon?

**Comment by victorhuangwq** [comment]
There's no update on this issue.

**Comment by champnic** [comment]
@akash07k We actually just took another look at this yesterday and will begin working on it in a couple weeks. We've already been working on the other accessibility issues you raised with JAWS/NVDA/Narrator and hope to have fixes for those soon.

**Comment by champnic** [comment]
This should be fixed in runtimes 128.0.2663.0+. Thanks!

**Comment by akash07k** [comment]
Thanks, when can we expect WebView2 runtime V128.0.2663.0?
> This should be fixed in runtimes 128.0.2663.0+. Thanks!

**Comment by champnic** [comment]
You can test now using a Canary or other prerelease runtime, or the stable 128 runtime will ship probably next week.

**Comment by akash07k** [comment]
Hmm, I tested it with Canary runtime just now and it is not yet fully fixed.
The tab navigation is still not working properly and pressing Tab navigates to only 1 element and then the navigation gets stucked. This issue should be reopened.
@champnic 
> You can test now using a Canary or other prerelease runtime, or the stable 128 runtime will ship probably next week.

**Comment by champnic** [comment]
@akash07k This issue was for alt+f4 and alt+space support. The support for full accessibility is in a pull request that we expect to be available for testing soon.

**Comment by akash07k** [comment]
Ok. will wait for the fix
> @akash07k This issue was for alt+f4 and alt+space support. The support for full accessibility is in a pull request that we expect to be available for testing soon.
